README.md

```diff
@@ -38,6 +38,19 @@ psql "port=9900 host=localhost password=peerdb"
 
 <img src="images/peerdb-demo.gif" width="512" />
 
+### **IMPORTANT: Ensuring ClickHouse Access to MinIO**
+
+If your ClickHouse DB runs outside Docker (e.g., on VMs or ClickHouse Cloud), it may not access MinIO, which is used to stage files before loading them. Ensure ClickHouse has network access to MinIO.
+
+PeerDB stages PostgreSQL data in MinIO within the Docker stack. Since ClickHouse is outside Docker, it needs a resolvable hostname for MinIO.
+
+Update `docker-compose.yml` and set `AWS_ENDPOINT_URL_S3` to MinIO's accessible IP:
+```yaml
+AWS_ENDPOINT_URL_S3: http://172.31.26.57:9001
+```
```

Comment:
```suggestion
Update `docker-compose.yml` and set `AWS_ENDPOINT_URL_S3` to MinIO's accessible IP (from both PeerDB and ClickHouse):
```yaml
AWS_ENDPOINT_URL_S3: http://172.31.26.57:9001 # Change this to IP/host which is accessible by both PeerDB and ClickHouse
```
```